workshop/README.md

```diff
@@ -0,0 +1,37 @@
+# An introduction to Kubernetes Service Catalog
+
+[Service Catalog](https://github.com/kubernetes-incubator/service-catalog) lets you provision cloud services and attach them to running Kubernetes deployments 
+without having to manage those resources directly. This is done through the use of a broker, a web
+server that implements the [Open Service Broker API](https://www.openservicebrokerapi.org/).
+
+# Objectives
+These labs are a introduction to the installation and use of Service Catalog inside a Kubernetes cluster. By the end of this course, you'll:
+* Understand the core concepts of the Open Service Broker API
+* Deploy an application with in memory state
+* Manually deploy a backing store
+* Manually attach the backing store to your application
+* Deploy a service broker, attach it to your Kubernetes deployment, and use it to automatically provision a backing store
+* Bind this OSB-provisioned backing store to your application running on Kubernetes
+
+# Prerequesites
+* Have completed [Kube101](https://github.com/IBM/kube101/tree/master/workshop).
+* Have a running Kubernetes cluster as per Kube101
+* A free-tier [IBM Cloud account](https://console.bluemix.net/registration/)
```

Comment:
this last bullet seems like a dup of the previous one.

Reply:
do you mean that having a free-tier account is superfluous given the previous bullet, or that I literally reiterate that somewhere?

Reply:
superfluous with previous bullet. If they have running Kube per Kube101 then that's all that matters. Right?